test/Prefixer.js

```diff
@@ -271,4 +271,8 @@ describe('Prefixer', () => {
 	test('tab-size', () => {
 		expect(prefix(`tab-size:1;`, 8)).to.equal([`-moz-tab-size:1;`, `tab-size:1;`].join(''))
 	})
+
+	test('css variables', () => {
+		expect(prefix(`--CircularProgress-animation:0.5s linear;`, 28)).to.equal('--CircularProgress-animation:0.5s linear;')
```

Comment:
Without this change we were accidentally hitting this branch of the code for this input:
https://github.com/thysultan/stylis/blob/08b179da63fe6f06ffc2ca9a166bbc5003366c33/src/Prefixer.js#L21-L23